app/helpers/template_helpers.py

```diff
@@ -131,6 +131,7 @@ def get_footer_urls(language_code: str, base_url: str, schema_theme: str):
         accessibility_statement_path = "hygyrchedd/"
         privacy_and_data_protection_path = "preifatrwydd-a-diogelu-data/"
         terms_and_conditions_path = "telerau-ac-amodau/"
+        ons_logo_language = "cy"
```

Comment:
I think this should just be passed as `language` which can be used for anything that needs language.